consensus/polybft/bridge_manager.go

```diff
@@ -358,8 +363,9 @@ func (b *bridgeManager) AddLog(chainID *big.Int, eventLog *ethgo.Log) error {
 
 // InsertEpoch inserts a new epoch to db with its meta data
 func (b *bridgeManager) InsertEpoch(epochNumber uint64, dbTx *bolt.Tx) error {
-	if err := b.state.EpochStore.insertEpoch(epochNumber, dbTx, b.chainID); err != nil {
-		return fmt.Errorf("an error occurred while inserting new epoch in db, chainID: %d. Reason: %w", b.chainID, err)
+	if err := b.state.EpochStore.insertEpoch(epochNumber, dbTx, b.externalChainID); err != nil {
```

Comment:
I think we need to insert epoch for the internal chain as well, but I wouldn't do that here, because it will mean that `n` number of `bridgeManagers` would insert it.
Add that to the `bridge` struct and its `PostEpoch` function.